docs/STYLEGUIDE.md

```diff
@@ -7,6 +7,7 @@ All Gitcoin UI styled classes are prefixed with `g-` to distincly identify them.
 
 - `g-font-muli` (Muli - Default)
 - `g-font-futura` (Futura - For Marketing Pages)
+- `g-multiselect` (For multiple tags style)
```

Comment:
This would not be under `Typography`

This would be under a new section called

```
## Forms

- *Multiselect* : `g-multiselect`

Usage
```<code-snippet>```